ci-operator/config/openshift-knative/kn-plugin-func/openshift-knative-kn-plugin-func-release-next.yaml

```diff
@@ -22,6 +25,26 @@ tests:
   commands: GOFLAGS="" make test
   container:
     from: src
+- as: e2e-oncluster-test
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: serverless-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.15"
```

Comment:
The components on [this line](https://github.com/openshift/release/pull/51325/files#diff-ea4238468a197a400dd9812ad0f0865982a9343ef66b90955a20f5aa8d0dcae9R15) are 4.16 but this cluster pool is 4.15. 
Did you want to use 4.15 components? It's hard to understand why they differ.

Reply:
ok I just removed the previous sections as you explained me ;) thanks